### ISSUE TITLE
Unmet dependencies warning during yarn install

### ISSUE THREAD

**Issue by filippsen** [SuperblocksHQ/superblocks-lab#24]
## Environment/Browser
* node v10.5.0
* yarn 1.7.0
* npm 6.1.0
* Or simply via Docker image: `node:latest` (8753edeb1aa3)

## Description
Unmet (missing?) dependencies warning during package installation.
The issue apparently hasn't been an offender in practice. 
The warning is only flagged by `yarn`.

## Steps to reproduce
`yarn install`

## Expected result
Nothing.

## Actual result
```
warning " > @fortawesome/react-fontawesome@0.0.17" has unmet peer dependency "prop-types@^15.5.10".
warning " > @fortawesome/react-fontawesome@0.0.17" has unmet peer dependency "react@^16.0.0".
warning " > less-loader@4.1.0" has unmet peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
warning " > react-fetch@0.0.9" has unmet peer dependency "react@>=0.14.0 || >15.0.0".
warning " > react-monaco-editor@0.13.0" has unmet peer dependency "react@^15.x || ^16.x".
```

## Reproducible
Can be consistently reproduced.

**Comment by filippsen** [comment]
Same happens when doing `npm install`, just with a slightly different message:

```
npm WARN @fortawesome/react-fontawesome@0.0.17 requires a peer of react@^16.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-fetch@0.0.9 requires a peer of react@>=0.14.0 || >15.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-monaco-editor@0.13.0 requires a peer of react@^15.x || ^16.x but none is installed. You must install peer dependencies yourself.
```

**Comment by filippsen** [comment]
Additional dependencies were added since then:  

```
warning " > react-onclickoutside@6.7.1" has unmet peer dependency "react@^15.5.x || ^16.x".
warning " > react-onclickoutside@6.7.1" has unmet peer dependency "react-dom@^15.5.x || ^16.x".

warning " > react-redux@5.0.7" has unmet peer dependency "react@^0.14.0 || ^15.0.0-0 || ^16.0.0-0".

warning " > babel-jest@23.4.2" has unmet peer dependency "babel-core@^6.0.0 || ^7.0.0-0".

warning " > babelify@8.0.0" has unmet peer dependency "babel-core@6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc".

warning "preact-cli > babel-loader@7.1.4" has unmet peer dependency "babel-core@6".
```

**Comment by javier-tarazaga** [comment]
Solved in #210
